server/functions.lua

```diff
@@ -173,40 +164,37 @@ function QBCore.Functions.GetEntitiesInBucket(bucket --[[int]])
     end
 end
 
---- Will return true / false wheter the mentioned player id is present in the bucket provided
-function QBCore.Functions.IsPlayerInBucket(player_source --[[int]] ,bucket --[[int]])
-    local curr_player_bucket = GetPlayerRoutingBucket(player_source)
-    return curr_player_bucket == bucket
-end
-
 -- Paychecks (standalone - don't touch)
 
-function PaycheckLoop()
-    local Players = QBCore.Functions.GetQBPlayers()
-    for _, Player in pairs(Players) do
-        local payment = Player.PlayerData.job.payment
-        if Player.PlayerData.job and payment > 0 and (QBShared.Jobs[Player.PlayerData.job.name].offDutyPay or Player.PlayerData.job.onduty) then
-            if QBCore.Config.Money.PayCheckSociety then
-                local account = exports['qb-bossmenu']:GetAccount(Player.PlayerData.job.name)
-                if account ~= 0 then -- Checks if player is employed by a society
-                    if account < payment then -- Checks if company has enough money to pay society
-                        TriggerClientEvent('QBCore:Notify', Player.PlayerData.source, Lang:t('error.company_too_poor'), 'error')
+function PaycheckInterval()
+    if next(QBCore.Players) then
+        for _, Player in pairs(QBCore.Players) do
+            if Player then
+                local payment = Player.PlayerData.job.payment
+                if Player.PlayerData.job and payment > 0 and (QBShared.Jobs[Player.PlayerData.job.name].offDutyPay or Player.PlayerData.job.onduty) then
+                    if QBCore.Config.Money.PayCheckSociety then
+                        local account = exports['qb-bossmenu']:GetAccount(Player.PlayerData.job.name)
```

Comment:
```suggestion
                        local account = exports['qb-management']:GetAccount(Player.PlayerData.job.name)
```
When are we switching this over?

Reply:
Not yet as it's not in the recipe as of now, qb-bossmenu still is